actions/local_storage.jsx

```diff
@@ -18,3 +20,16 @@ export function setPreviousTeamId(teamId) {
         return {data: true};
     };
 }
+
+export function setPreviousRecentEmojis() {
+    return (dispatch, getState) => {
+        const currentUserId = getCurrentUserId(getState());
+
+        const recentEmojis = getState().storage.storage[Constants.RECENT_EMOJI_KEY];
+        if (recentEmojis && recentEmojis.value) {
+            LocalStorageStore.setPreviousRecentEmojis(currentUserId, recentEmojis.value);
+        }
+
+        return {data: true};
+    };
+}
```

Comment:
Attempted to add unit test but can't find a way this time to effectively make it.